### ISSUE TITLE
[Bug] Incompatible eslint-plugin-jsx-a11y version?

### ISSUE THREAD

**Issue by wassim** [roots/sage#1805]
## Submit a feature request or bug report

- [X] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/guidelines/blob/master/CONTRIBUTING.md)
- [ ] This is a feature request
- [x] This is a bug report
- [X] This request isn't a duplicate of an [existing issue](https://github.com/roots/sage/issues)
- [X] I've read the [docs](https://roots.io/sage/docs) and [NPM Debugging Guidelines post](https://discourse.roots.io/t/npm-debugging-guidelines-failed-npm-install-bower-install-or-gulp-build-read-this/3060) and followed them (if applicable)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/c/sage) forums

## Bug report

**Please provide steps to reproduce, including full log output:**

1. composer create-project roots/sage my-theme dev-master
2. yarn
3. change theme name in style.css
4. update publicPath and devUrl in config.json
5. yarn run start

- Terminal prints browser-sync errors
- Doesn't opens localhost:3000
- When localhost:3000 is manually typed, browser prints the following:

```
Module build failed: Error: /Users/wassim/Projects/sites/mysite.com/site/web/app/themes/mysite/node_modules/eslint-config-airbnb/rules/react-a11y.js:
	Configuration for rule "jsx-a11y/anchor-has-content" is invalid:
	Value "[object Object]" no (or more than one) schemas match.
```

**Please describe your local environment:**

WordPress version: 4.7

Sage: 9.0.0-beta.1

OS: macOS Sierra

NPM/Node version: 4.0.5 / 6.9.4

**Where did the bug happen? Development or remote servers?**

Development

**Comment by retlehs** [comment]
closed by #1796
